src/main/resources/messages.properties

```
@@ -194,6 +194,13 @@ children-ccap-in-care.header=Is {0} already in child care with your chosen provi
 #activities-ed-program-type
 #activities-ed-program-name
 #activities-ed-program-info
+activities-ed-program-info.header=Program information
+activities-ed-program-info.description=You can skip questions that are not relevant to your program.
+activities-ed-program-info.phone-number=What is their phone number?
```

Copy is slightly different in the figma
```suggestion
activities-ed-program-info.phone-number=What's their phone number?
```

---

I follow the pattern of other fields. Should I follow exactly what the figma looks like?

---

Yeah generally - when Carlie gets back, we can ask her if that should be changed